### DISCUSSION TITLE
Can't get Bluetooth mac address to pair with PS3 controller

### DISCUSSION THREAD

**Discussion by RedCast08** [darthcloud/BlueRetro#362] [General]
Hi, i'm trying to pair it with a PS3 controller and I can't find the Mac address.
![Capture](https://user-images.githubusercontent.com/99358504/185462085-404f4211-9fb4-4dfc-bc20-9d35030cb3d9.JPG)

**Comment by darthcloud** [comment]
Update your BlueRetro FW to v1.7.1

**Comment by RedCast08** [comment]
I tried to update the device, I selected the zip file and the upload didn't update it. I took over 30mins for it upload.

**Reply by darthcloud** [reply]
did you unzip it first and select the right file?

**Reply by RedCast08** [reply]
I could not find any documentation on OTA update? There were a lot of files in the zip. I just assumed the zip file was the firmware.

**Reply by darthcloud** [reply]
https://github.com/darthcloud/BlueRetro/wiki#via-web-bluetooth-interface-ota-fw-update

**Reply by darthcloud** [reply]
take the file name psx_external.bin

**Reply by RedCast08** [reply]
Thank you! I was looking for the BlueRetro*.bin. I have the PS3 controller paired now.

**Reply by Wild-Dog** [reply]
I'm not sure where to find this. I only see BlueRetro_hw1_playstation.bin in the zip.

Updated it and still no mac address on the system manager.

**Reply by emuelfe84** [reply]
Hello, I have the same problem. I can get the mac adress from the blue retro system manager to connet my ps3 controller. Can you help me darthcloud?

![image](https://github.com/user-attachments/assets/29f7a86a-3239-476b-9a55-9b484b545088)
